### ISSUE TITLE
Stop all sound

### ISSUE THREAD

**Issue by ariadesu** [fr33mind/Belle#19]
Regardless of which scene you transition from, you don't want two songs playing over each other. Turning off songs by song name is clunky

**Comment by fr33mind** [comment]
You're right. I can add a stop all sounds action, like you mentioned, or also add some option to the scene.
I'll add it in the next version, thanks.
